app/models/topic.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Topic
+  def self.govuk_homepage
+    find(TopicIndex::GOVUK_HOMEPAGE_CONTENT_ID)
+  end
+
+  def self.find(topic_content_id)
+    TopicIndex.new.index[topic_content_id]
+  end
+
+  attr_reader :title, :child_content_ids, :legacy_topic_content_ids, :parent_content_id, :content_id
+
+  def initialize(**params)
```

Comment:
Do we need a double splat here?

Reply:
Oops again - changed tack as I wrote the method. Fixed.